sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -3081,4 +3081,19 @@ class ColumnExpressionSuite extends QueryTest with SharedSparkSession {
     checkAnswer(df.select($"dd" / ($"num" + 3)),
       Seq((Duration.ofDays(2))).toDF)
   }
+
+  test("luhn check") {
```

Comment:
I think the golden file tests fully cover it. If we add a scala API for luhn check, we need this test.

Reply:
Done